operations/tasks/update-node_selector-labels.yml

```diff
@@ -0,0 +1,18 @@
+---
+# delete all predefined node selector labels
+- name: Delete all labels
+  command: >
+    oc label node {{ inventory_hostname }} {{ item }}-
+  with_items: {{ osbs_all_node_selector_labels }}
+  register: delete_label
+  failed_when: delete_label.rc != 0
```

Comment:
Not needed. This is the default behavior.

Reply:
right, will remove it